.gitignore

```diff
@@ -3,4 +3,5 @@
 *.ipr
 .idea
 /_site
-.jekyll-metadata
\ No newline at end of file
+.jekyll-metadata
+documentation/.DS_Store
```

Comment:
Line 7 is ignoring a Mac-specific file, and in this case we should make it more generic and replace `documentation/.DS_Store` with `**/.DS_Store`